2.4/s2i/bin/usage

```diff
@@ -1,18 +1,16 @@
 #!/bin/sh
 
-DISTRO=`cat /etc/*-release | grep ^ID= | grep -Po '".*?"' | tr -d '"'`
-NAMESPACE=centos
-[[ $DISTRO =~ rhel* ]] && NAMESPACE=rhscl
+DISTRO="`grep ^ID= /etc/*-release | cut -d'"' -f2`"
 
 cat <<EOF
-This is a S2I ruby-2.4 ${DISTRO} base image:
+This is a S2I ruby-${RUBY_VERSION} ${DISTRO} base image:
 To use it, install S2I: https://github.com/openshift/source-to-image
 
 Sample invocation:
 
-s2i build https://github.com/sclorg/s2i-ruby-container.git --context-dir=2.4/test/puma-test-app/ ${NAMESPACE}/ruby-24-${DISTRO}7 ruby-sample-app
+s2i build https://github.com/sclorg/s2i-ruby-container.git --context-dir=${RUBY_VERSION}/test/puma-test-app/ ${IMAGE_NAME} ruby-sample-app
 
 
 You can then run the resulting image via:
-docker run -p 8080:8080 ruby-sample-app
+podman run -p 8080:8080 ruby-sample-app
```

Comment:
This is intended ? What about for RHEL7 containers, there is no podman command. It is available only on RHEL8

Reply:
This is just a recommendation and 2.5+ has currently `podman` (without my change); I just wanted to unify it.